crates/cairo-coverage/src/output/lcov.rs

```diff
@@ -0,0 +1,105 @@
+use crate::intermediate_representation::{
+    FunctionDetails, FunctionDetailsOps, FunctionsByFile, FunctionsInFile, FunctionsInFileOps,
+};
+use crate::types::{FileLocation, FunctionName, HitCount, LineNumber};
+use derived_deref::Deref;
+use std::fmt;
+use std::fmt::Display;
+
+#[derive(Deref)]
+pub struct LcovFormat(Vec<(FileLocation, LcovData)>);
+
+pub struct LcovData {
+    lines: Vec<(LineNumber, HitCount)>,
+    hit_count: HitCount,
```

Comment:
What does it represent? all hits count in the file? `file_hit_count` seems like a better name then